### ISSUE TITLE
Incompatible with neovim stable due to use of lua-fs

### ISSUE THREAD

**Issue by bjornsnoen** [numToStr/prettierrc.nvim#1]
This module only works with vim nightly as it uses lua-fs in order to find the correct rc file. The offending code is [here](https://github.com/numToStr/prettierrc.nvim/blob/master/lua/prettierrc/init.lua#L94)

**Comment by numToStr** [comment]
Yeah, I knowingly used `vim.fs` which is nightly only feature (as I am a nightly user). I'll put requirement on README.

**Comment by bjornsnoen** [comment]
Alright, will this be a requirement for ever or will this be usable on stable from 0.8 going forward?

**Comment by numToStr** [comment]
Yes, `vim.fs` is technically a `v0.8` API so you need to wait for it to be released as stable or use nightly for now if you can :)

**Comment by bjornsnoen** [comment]
Yeah, I more meant will you try to not use new nightly only APIs going forward? Say once 0.8 is out, some new feature is requested; would you try to only use stable APIs to implement or can we expect the following version of the plugin to again only work on nightly?

**Comment by numToStr** [comment]
> will you try to not use new nightly only APIs going forward?

I also, almost, follow this principle unless something is really required like `vim.fs`. And the plugin is relatively stable, i hope, so I don't think any other nightly features will be required afterwards once 0.8 is released.
